anda/tools/rustypaste/rustypaste.spec

```diff
@@ -0,0 +1,65 @@
+%define __brp_mangle_shebangs %{nil}
+
+Name:          rustypaste
+Version:       0.16.1
+Release:       1%?dist
+Summary:       A minimal file upload/pastebin service
+License:       MIT
+URL:           https://github.com/orhun/rustypaste
+Source0:       %url/archive/refs/tags/v%{version}.tar.gz
+
+BuildRequires: anda-srpm-macros
+BuildRequires: cargo-rpm-macros
+BuildRequires: gcc
+BuildRequires: cargo
+BuildRequires: mold
+BuildRequires: systemd-rpm-macros
+
+Requires:      openssl
+Requires:      pulseaudio-libs
```

Comment:
can't find pulseaudio mentioned in upstream readme

for openssl remove this requires unless you add `-f openssl` to `%cargo_build` but anyhow that mandates `BuildRequires: pkgconfig(openssl)` anyway making this requires unnecessary regardless.